### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akiomik**
Add some bots, clients, and libraries.

### CODE REVIEW

**Review by mattn — APPROVED** [review]

**Comment by mattn** [comment]
Good catch. Thank you.

File: README.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ Awesome [nostr](https://nostr.com/) in Japan. Software, Web service, Clients, Bo
 * [golang_news](https://iris.to/golang_news@mattn.github.io) - [Go](https://go.dev) language news by [mattn](https://github.com/mattn)
 * [nips_changes](https://iris.to/nips_changes@mattn.github.io) - Git commit changes on [github.com/nostr-protocol/nips](https://github.com/nostr-protocol/nips/) by [mattn](https://github.com/mattn)
 * [makeitquote](https://iris.to/makeitquote) - Generate image of the post by [mattn](https://github.com/mattn)
+* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn)
+* [mahjong](https://nostx.shino3.net/npub1ttqyyl8stz9wtj0sn25qp6vah0jdcxwpdtaaxg4efsqkczz7rsxshjpp3x) - A bot for playing mahjong game by [mattn](https://github.com/mattn)
```
Comment (mattn) [inline README.md:68]:
```suggestion
* [mahjong](https://nostx.shino3.net/npub1ttqyyl8stz9wtj0sn25qp6vah0jdcxwpdtaaxg4efsqkczz7rsxshjpp3x) - A bot for playing mahjong game by [mattn](https://github.com/mattn)
* [mahjong](https://nostx.shino3.net/npub1ttqyyl8stz9wtj0sn25qp6vah0jdcxwpdtaaxg4efsqkczz7rsxshjpp3x) - A bot for playing mahjong game by [mattn](https://github.com/mattn/nostr-mahjongbot)
```

File: README.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ Awesome [nostr](https://nostr.com/) in Japan. Software, Web service, Clients, Bo
 * [golang_news](https://iris.to/golang_news@mattn.github.io) - [Go](https://go.dev) language news by [mattn](https://github.com/mattn)
 * [nips_changes](https://iris.to/nips_changes@mattn.github.io) - Git commit changes on [github.com/nostr-protocol/nips](https://github.com/nostr-protocol/nips/) by [mattn](https://github.com/mattn)
 * [makeitquote](https://iris.to/makeitquote) - Generate image of the post by [mattn](https://github.com/mattn)
```
Comment (mattn) [inline README.md:66]:
```suggestion
* [makeitquote](https://iris.to/makeitquote) - Generate image of the post by [mattn](https://github.com/mattn/nostr-makeitquote)
```

File: README.md [hunk 3]
```diff
@@ -64,6 +64,8 @@ Awesome [nostr](https://nostr.com/) in Japan. Software, Web service, Clients, Bo
...
+* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn)
```
Comment (mattn) [inline README.md:67]:
```suggestion
* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn/cloudflare-nostr-nullpoga)
```

File: README.md [hunk 4]
```diff
@@ -64,6 +64,8 @@ Awesome [nostr](https://nostr.com/) in Japan. Software, Web service, Clients, Bo
 * [golang_news](https://iris.to/golang_news@mattn.github.io) - [Go](https://go.dev) language news by [mattn](https://github.com/mattn)
 * [nips_changes](https://iris.to/nips_changes@mattn.github.io) - Git commit changes on [github.com/nostr-protocol/nips](https://github.com/nostr-protocol/nips/) by [mattn](https://github.com/mattn)
 * [makeitquote](https://iris.to/makeitquote) - Generate image of the post by [mattn](https://github.com/mattn)
+* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn/cloudflare-nostr-nullpoga)
+* [mahjong](https://nostx.shino3.net/npub1ttqyyl8stz9wtj0sn25qp6vah0jdcxwpdtaaxg4efsqkczz7rsxshjpp3x) - A bot for playing mahjong game by [mattn](https://github.com/mattn/nostr-mahjongbot)
```
Comment (mattn) [inline README.md:68]:
```suggestion
* [mahjong](https://nostx.shino3.net/npub1ttqyyl8stz9wtj0sn25qp6vah0jdcxwpdtaaxg4efsqkczz7rsxshjpp3x) - A bot for playing mahjong game by [mattn](https://github.com/mattn)
```

File: README.md [hunk 5]
```diff
@@ -64,6 +64,8 @@ Awesome [nostr](https://nostr.com/) in Japan. Software, Web service, Clients, Bo
 * [golang_news](https://iris.to/golang_news@mattn.github.io) - [Go](https://go.dev) language news by [mattn](https://github.com/mattn)
 * [nips_changes](https://iris.to/nips_changes@mattn.github.io) - Git commit changes on [github.com/nostr-protocol/nips](https://github.com/nostr-protocol/nips/) by [mattn](https://github.com/mattn)
 * [makeitquote](https://iris.to/makeitquote) - Generate image of the post by [mattn](https://github.com/mattn)
+* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn/cloudflare-nostr-nullpoga)
```
Comment (mattn) [inline README.md:67]:
```suggestion
* [nullpoga](https://nostx.shino3.net/npub1f6rvmwc76arl7sxx2vparlzx8cg2ajc3xpymqh7yx97znccue2hs5mkavc) - A misc bot by [mattn](https://github.com/mattn)
```

**Comment by mattn** [comment]
Still working on this?

**Comment by akiomik** [comment]
> Still working on this?

I will add a little more.

**Comment by akiomik** [comment]
> > Still working on this?
> 
> Yes, I will add a little more.

Added ✅

**Review by SnowCait — APPROVED** [review]
🎉

**Review by nikolat — APPROVED** [review]
🙇

**Review by mattn — APPROVED** [review]

**Review by awayuki — APPROVED** [review]
LGTM 🤙

**Review by erechorse — APPROVED** [review]

**Comment by mattn** [comment]
Thanks all.

**Pull request merged**